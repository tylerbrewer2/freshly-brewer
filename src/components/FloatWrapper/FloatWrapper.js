import styled from 'styled-components';

const FloatWrapper = styled.div`
  float: ${props => props.direction}
`

export default FloatWrapper;
