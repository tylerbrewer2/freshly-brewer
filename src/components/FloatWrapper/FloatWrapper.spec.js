import React from 'react';
import renderer from 'react-test-renderer';

import FloatWrapper from './FloatWrapper';

it('renders correctly', () => {
  const tree = renderer.create(
    <FloatWrapper />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
