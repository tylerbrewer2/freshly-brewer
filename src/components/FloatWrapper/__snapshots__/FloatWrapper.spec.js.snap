exports[`test renders correctly 1`] = `
<div
  className="sc-bdVaJa iCWnlt" />
`;
